### PR TITLE
br: remove --tikv-max-restore-concurrency

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Remove the ` --tikv-max-restore-concurrency` parameter from v7.6.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16353 and https://github.com/pingcap/docs-cn/pull/16348
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

File: releases/release-7.6.0.md [hunk 1]
```diff
@@ -69,16 +69,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.6/quick-start-with-
...
 
-    The new coarse-grained Region scatter algorithm is experimental. To use it, you can configure the `--granularity="coarse-grained"` parameter in the `br` command. You can also control the concurrency of download tasks on each TiKV node by setting the `--tikv-max-restore-concurrency` parameter. For example:
+    The new coarse-grained Region scatter algorithm is experimental. To use it, you can configure the `--granularity="coarse-grained"` parameter in the `br` command. For example:
 
     ```bash
     br restore full \
     --pd "${PDIP}:2379" \
     --storage "s3://${Bucket}/${Folder}" \
     --s3.region "${region}" \
     --granularity "coarse-grained" \
```
Comment (benmaoer) [inline releases/release-7.6.0.md:79]:
```suggestion
    --granularity "coarse-grained" \
    --send-credentials-to-tikv=true \    
```

**Review by benmaoer — CHANGES_REQUESTED** [review]
keep --send-credentials-to-tikv=true

File: releases/release-7.6.0.md [hunk 2]
```diff
@@ -69,16 +69,15 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.6/quick-start-with-
...
-    --tikv-max-restore-concurrency 128 \
-    --send-credentials-to-tikv=true \
+    --send-credentials-to-tikv=true \
```
Comment (Oreoxmt) [inline releases/release-7.6.0.md:80]:
```suggestion
    --send-credentials-to-tikv=true \ 
```

**Review by benmaoer — APPROVED** [review]
LGTM

**Review by 3pointer — APPROVED** [review]

**Pull request merged**